### PR TITLE
fix(PPT-1540): Update settings with proper names

### PR DESCRIPTION
**Pull request by grkek**

### CODE REVIEW

**Review by stakach — COMMENTED** [review]
this is a breaking change though... we'd need to investigate the impact on clients who have the old setting configured

**Comment by grkek** [comment]
> this is a breaking change though... we'd need to investigate the impact on clients who have the old setting configured

I can put in a clause which will handle both of these

File: drivers/place/bookings.cr [hunk 1]
```diff
@@ -32,6 +32,7 @@ class Place::Bookings < PlaceOS::Driver
 
     application_permissions:    true,
     include_cancelled_bookings: false,
+    show_qr_code:               false,
     hide_qr_code:               false,
```
Comment (stakach) [inline drivers/place/bookings.cr:36]: We should remove the old setting from the defaults
Otherwise looks good

**Review by stakach — CHANGES_REQUESTED** [review]

File: drivers/place/bookings.cr [hunk 2]
```diff
@@ -164,7 +164,11 @@ class Place::Bookings < PlaceOS::Driver
 
     self[:custom_qr_color] = setting?(String, :custom_qr_color)
     self[:custom_qr_url] = setting?(String, :custom_qr_url)
-    self[:show_qr_code] = !(setting?(Bool, :hide_qr_code) || false)
+
+    hide_qr_code = setting?(Bool, :hide_qr_code) || false
+    show_qr_code = setting?(Bool, :show_qr_code) || false
+
+    self[:show_qr_code] = !hide_qr_code || show_qr_code
```
Comment (stakach) [inline drivers/place/bookings.cr:171]: The settings both kind of clash
i.e. by default hide_qr_code == true, but if show_qr_code is set to false then the setting won't be applied.

```suggestion
    self[:show_qr_code] = show_qr_code.nil? ? !hide_qr_code : show_qr_code
```

File: drivers/place/bookings.cr [hunk 3]
```diff
@@ -164,7 +164,11 @@ class Place::Bookings < PlaceOS::Driver
 
     self[:custom_qr_color] = setting?(String, :custom_qr_color)
     self[:custom_qr_url] = setting?(String, :custom_qr_url)
-    self[:show_qr_code] = !(setting?(Bool, :hide_qr_code) || false)
+
+    hide_qr_code = setting?(Bool, :hide_qr_code) || false
+    show_qr_code = setting?(Bool, :show_qr_code) || false
```
Comment (stakach) [inline drivers/place/bookings.cr:169]:
```suggestion
    show_qr_code = setting?(Bool, :show_qr_code)
```

**Review by stakach — CHANGES_REQUESTED** [review]
needs to be backwards compatible and the settings can't clash

**Pull request merged**